src/mutation.h

```diff
@@ -114,6 +114,9 @@ struct mutation_branch {
     // Modifier for the rate at which fatigue drops when resting.
     float fatigue_regen_modifier = 0.0f;
 
+    /** The item, if any, spawned by the mutation */
+    std::string spawn_item;
```

Comment:
yes, it does, because `itype_id` is a dumb `typedef` for `std::string`, but hopefully not for long. `itype_id` will become an alias for `string_id<itype_t>`. For now we just try to use `itype_id` where it's meant to be.